scripts/zones/Windurst_Woods/npcs/Perih_Vashai.lua

```diff
@@ -7,18 +7,19 @@
 -----------------------------------
 package.loaded["scripts/zones/Windurst_Woods/TextIDs"] = nil;
 -----------------------------------
-
+require("scripts/zones/Windurst_Woods/TextIDs");
 require("scripts/globals/settings");
 require("scripts/globals/titles");
 require("scripts/globals/keyitems");
 require("scripts/globals/quests");
-require("scripts/zones/Windurst_Woods/TextIDs");
 require("scripts/globals/missions");
+
+local twinstoneEarring = 13360;
+local oldEarring = 1113;
```

Comment:
Its used in multiple functions. If you move it you break one. Previously the item I'ds were used but somebody merged the change to putting them like this and didn't make the previous person declare them twice (once in each function). If this isn't acceptable then somebody screwed up code review when the prior change got merged.

I'm making this more readable. Do you want it declared per function, or go back to using the item IDs again, instead of this?